nixos/modules/services/hardware/udev.nix

```diff
@@ -8,6 +8,24 @@ let
 
   cfg = config.services.udev;
 
+  initrdUdevRules = pkgs.runCommand "initrd-udev-rules" {} ''
```

Comment:
In hind-sight I do not think this is doing what you think it's doing.


systemd by default loads all udev rules in `$out/lib/systemd/udev` so this just includes all the rules twice in the initrd. Not limit what rules get loaded.